lib/assessment.rb

```diff
@@ -20,21 +23,24 @@ def sum(enum)
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    if numbers == odd?
+      return true
   end
+    end
```

Comment:
You have to use `any?` in conjunction with `odd?`. `odd?` only checks if any given number is odd.
`numbers.any?(&:odd?)`